projectile.el

```diff
@@ -1050,7 +1049,7 @@ discover projects there."
           (dolist (dir (directory-files directory t))
             (when (and (file-directory-p dir)
                        (not (member (file-name-nondirectory dir) '(".." "."))))
-              (projectile-discover-projects-in-directory dir (1- depth))))
+              (projectile-discover-projects-in-directory dir)))
```

Comment:
Why did you change this?

Reply:
You're right. This shouldn't be here. Reverted.